types/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    // Use prettier for formating
```

Comment:
I think that comments like these are not actually valid JSON (big shame thogh)

Reply:
You're right, in plain JSON it's not possible (Nice short read from Douglas Crockford on [G+](https://plus.google.com/+DouglasCrockfordEsq/posts/RK8qyGVaGSr)). But because it's useful there are several JSON parser that can skip this and e.g. tslint, eslint, and tsc are using them.

Reply:
I'm not sure about the usefulness of the comment tho, is it because we disable rules that prettier does things with? Maybe use https://www.npmjs.com/package/tslint-config-prettier since it automatically turns off all rules that prettier does things with